examples/stamped_bin/stamped_bin_test.go

```diff
@@ -22,7 +22,13 @@ import (
 )
 
 func TestStampedBin(t *testing.T) {
-	if stamp.BUILD_TIMESTAMP == stamp.NOT_A_TIMESTAMP {
+	if stamp.BUILD_TIMESTAMP == stamp.NOT_A_TIMESTAMP || stamp.BUILD_TIMESTAMP == "" {
```

Comment:
When overridden with a stamped `x_def` (e.g. [here](https://github.com/ixdy/bazel-rules_go/blob/30d98e0cb710a10e04655330d73c1b150fa3fda5/examples/stamped_bin/BUILD#L18)) if the variable doesn't exist or the implementation doesn't work (which is what happened to me first. :))

This is mostly covered by other cases now, so I can revert this if you'd like.